src/nunavut/lang/c/templates/_fields.j2

```diff
@@ -0,0 +1,11 @@
+{%- for field in composite_type.fields %}
+{%- if field is not padding %}
```

Comment:
There is `fields_except_padding` just for this. Applies to the union as well.

Reply:
Oh nice!